Sources/SCTE35/HelperConverter/BitConverter.swift

```diff
@@ -67,7 +146,7 @@ class BitConverter {
         return string
     }
 
-    static func smpteString(fromBits bits: [Bit]) -> String? {
+    static func umidString(fromBits bits: [Bit]) -> String? {
```

Comment:
SMPTE is an entity, the actual string being parsed is an UMID string.  Thus I changed the name of this method.